### DISCUSSION TITLE
ListView is not rendered

### DISCUSSION THREAD

**Discussion by DerTev** [tui-cs/Terminal.Gui#2774] [Q&A] [answered]
Any idea, why the `ListView` is not rendered?
As you can see, the List contains an element.

```
        foreach (var step in Steps!) StepKeys.Add(step.Key);

        Title = "ABC";

        var length = new Label("Length: " + StepKeys.Count);

        var stepSelection = new ListView(StepKeys);

        var exitButton = new Button
        {
            Text = "Exit",
            X = Pos.Left (stepSelection),
			Y = Pos.Bottom (stepSelection) + 1
        };

        Add(length, stepSelection, exitButton);
    }
```

<img width="80" alt="image" src="https://github.com/gui-cs/Terminal.Gui/assets/77175275/ff10af4a-8edf-469c-8a0f-b42aad3bb8da">

**Comment by tig** [comment, marked as answer]
Most likely this is because you have not specified a size for the ListView.

Try:

```cs
var stepSelection = new ListView(StepKeys) { Y = Pos.Bottom(length), Height = 5, Width = 20 };
```

**Comment by DerTev** [comment]
Ah, that works, thanks!

Is Height here the height of items that would fit in the height of the element?

**Reply by tig** [reply]
Height of element.
